testworkload/nginx_workload.go

```diff
@@ -58,46 +58,36 @@ func NewNginx(kc kubernetes.Interface, namespace string, options ...NginxOpts) (
 		},
 		client: kc,
 	}
-
 	//apply options
 	for _, option := range options {
 		if err := option(n); err != nil {
 			return nil, fmt.Errorf("error invalid options: %v", err)
 		}
 	}
 
-	//create nginx deployment
-	if err := n.newNginxDeployment(); err != nil && !apierrors.IsAlreadyExists(err) {
-		return nil, fmt.Errorf("error creating deployment %s: %v", n.Name, err)
-	}
 	if err := wait.PollImmediate(PollIntervalForNginx, PollTimeoutForNginx, func() (bool, error) {
+		//create nginx deployment
```

Comment:
this comment is kind of redundant, the next line says exactly the same thing (same with `newNginxService()`)